src/RefElemData.jl

```diff
@@ -180,6 +180,7 @@ RefElemData(elem, N::Int; kwargs...) = RefElemData(elem, Polynomial(), N; kwargs
 @inline face_type(::Union{Tri, Quad}) = Line()
 @inline face_type(::Hex) = Quad()
 @inline face_type(::Tet) = Tri()
+@inline face_type(::Wedge, id) = (id < 3) ? Quad() : Tri()
```

Comment:
```suggestion
@inline face_type(elem::AbstractElemShape, id) = face_type(elem)
@inline face_type(::Wedge, id) = (id < 3) ? Quad() : Tri()
```
Good idea. I'd suggest adding a general fallback so the other elements also have the same interface.